daprdocs/content/en/getting-started/quickstarts/jobs-quickstart.md

```diff
@@ -0,0 +1,223 @@
+- The `C-3PO` job is being retrieved.
+- The `BB-8` job is being scheduled.
+- The `BB-8` job is being retrieved.
+- The `BB-8` job is being deleted.
+- The `R2-D2` job is being executed after 5 seconds.
+- The `R2-D2` job is being executed after 10 seconds.
+
+**Expected output**
+
+```text
+== APP - job-service == dapr client initializing for: 127.0.0.1:6281
+== APP - job-service == Registered job handler for:  R2-D2
+== APP - job-service == Registered job handler for:  C-3PO
+== APP - job-service == Registered job handler for:  BB-8
+== APP - job-service == Starting server on port: 6200
+== APP - job-service == Job scheduled:  R2-D2
+== APP - job-service == Job scheduled:  C-3PO
+== APP - job-service == 2024/07/17 18:09:59 job:{name:"C-3PO"  due_time:"10s"  data:{value:"{\"droid\":\"C-3PO\",\"Task\":\"Memory Wipe\"}"}}
+== APP - job-scheduler == Get job response:  {"droid":"C-3PO","Task":"Memory Wipe"}
+== APP - job-service == Job scheduled:  BB-8
+== APP - job-service == 2024/07/17 18:09:59 job:{name:"BB-8"  due_time:"15s"  data:{value:"{\"droid\":\"BB-8\",\"Task\":\"Internal Gyroscope Check\"}"}}
+== APP - job-scheduler == Get job response:  {"droid":"BB-8","Task":"Internal Gyroscope Check"}
+== APP - job-scheduler == Deleted job:  BB-8
+```
+
+After 5 seconds, the terminal output should present the `R2-D2` job being processed:
+
+```text
+== APP - job-service == Starting droid: R2-D2
+== APP - job-service == Executing maintenance job: Oil Change
+```
+
+After 10 seconds, the terminal output should present the `C3-PO` job being processed:
+
+```text
+== APP - job-service == Starting droid: C-3PO
+== APP - job-service == Executing maintenance job: Memory Wipe
+```
+
+Once the process has completed, you can stop and clean up application processes with a single command.
+
+```bash
+dapr stop -f .
+```
+
+### What happened?
+
+{{% /codetab %}}
+
+{{< /tabs >}}
+
+## Run one job application at a time
+
+{{< tabs Go >}}
+
+ <!-- Go -->
+{{% codetab %}}
+
+This quickstart includes two apps:
+
+- **`job-scheduler.go`:** schedules, retrieves, and deletes jobs.
+- **`job-service.go`:** handles the scheduled jobs.
+
+### Step 1: Pre-requisites
+
+For this example, you will need:
+
+- [Dapr CLI and initialized environment](https://docs.dapr.io/getting-started).
+- [Latest version of Go](https://go.dev/dl/).
+<!-- IGNORE_LINKS -->
+- [Docker Desktop](https://www.docker.com/products/docker-desktop)
+<!-- END_IGNORE -->
+
+### Step 2: Set up the environment
+
+Clone the [sample provided in the Quickstarts repo](https://github.com/dapr/quickstarts/tree/master/jobs).
+
+```bash
+git clone https://github.com/dapr/quickstarts.git
+```
+
+From the root of the Quickstarts directory, navigate into the jobs directory:
+
+```bash
+cd jobs/go/sdk
+```
+
+### Step 3: Schedule jobs
+
+In the terminal, run the `job-service` app:
+
+```bash
+dapr run --app-id job-service --app-port 6200 --dapr-http-port 6280 --dapr-grpc-port 6281 --app-protocol grpc -- go run .
```

Comment:
Let's review the code example, since I do not think all these parameters are needed.

Reply:
the http-port can be removed.